### DISCUSSION TITLE
Newegg Bot Discussion

### DISCUSSION THREAD

**Discussion by TreborNamor** [TreborNamor/BestbuyBot#40] [General]
Welcome to the Newegg discussion board!

**Comment by Tr1cky1** [comment]
I tried testing on Newegg and it seems it is getting hung up where it has to enter CVV. I get the error it was unable. On the screen I can see where it hangs up it wants me to choose which shipping option I want (IE: Free, etc). Any ideas?

**Comment by Waqar8787** [comment]
TreborNamor bro
i need bot for
(United Arab Emirates)  https://www.newegg.com/global/ae-en/ please please

**Comment by Kali123411** [comment]
Is Newegg even doing drops any more? All I see is their daily shuffle you have to enter...

**Reply by Tr1cky1** [reply]
Yes. For the items in that days shuffle that I'm guessing weren't purchased. Usually around 10pm cst

**Reply by Kali123411** [reply]
cool thanks

**Comment by TheD0nald** [comment]
Is the newegg bot working as of right now? @TreborNamor I can do all this in powershell if you would like to colab a little. I know how to parse the html attributes etc.

**Comment by TheD0nald** [comment]
I have 90% of this built in powershell using selenium but I havent seen the best buy que system and need to test getting through that

Does anyone know if newegg requires any weird things to check out like best buy does?

**Comment by pyrokidd89** [comment]
The Newegg bot seems to fail when going to the cart there is a pop up that the bot needs to close before it can check if the item is still in the cart. I tried selecting don't show this again but it keeps coming back. can someone update the bot?
![image](https://user-images.githubusercontent.com/44952266/116705449-8c5e9a80-a99a-11eb-81e2-8eebc9630a42.png)

**Reply by TreborNamor** [reply]
I know how to fix this issue. More than likely you are using the webpage that the bot popped up for you. What you need to do is close all firefox webpages. Open a firefox window manually, and go to the shopping cart. You'll have to click "do not show this message again" and "im not interested". Now that you have done that. Firefox will remember your choice. It won't remember your choice when the bot opens up the webpage for some reason.

**Comment by TreborNamor** [comment]
You can find the bestbuy queue system with the ryzen 5800x. That's how I
tested the bot with queue system.

**Comment by latao3450** [comment]
@TreborNamor Hey man! First of all thank you for all your work and sharing. You are amazing.

I have adapted the bot a bit (learning python as you go haha) to work on the Canadian New Egg website. It worked fine with a test purchase. Only problem is when I run like bots on the gpus I want I got a "Are you human" prompt. Do you know any way around that?

**Comment by cmeept** [comment]
I have 1 problem with price_limit . It's not working .
I choose the limit of 600 and he still buys what is there, even if it costs 2000
How I can to fix it ? 
Thanks

**Comment by kangjiajinlong** [comment]
Newegg adds an additional step of choosing delivery options before final checkout. As a result, the bot seems outdated and is stuck at the step of filling in CVV. Could you kindly look at this issue and updated the script?

Thank you
